### PR TITLE
ENH: Add Drexel University

### LINKED ISSUE #47: Materials Science Seminar, Drexel University

### PR DESCRIPTION
**Pull request by dragonyanglong**
Close #47

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -420,4 +420,12 @@ uduisburgessen:
   state: North Rhine-Westphalia
   street: Universitaetsstrasse 7
   zip: 45117
-
+drexelu:
```
Comment (sbillinge) [inline db/institutions.yml]: Drexel is already an entry but it needs to have departments and schools added.  Please could you rebase and then do this over?  Thanks

**Comment by dragonyanglong** [comment]
Hi @sbillinge , with @chiahaoliu 's help, I edits the existing drexel u entry and did the rebase successfully. please could you review and merge? Thanks.

**Comment by dragonyanglong** [comment]
I just rebased few minutes ago. Should I redo it again? Or did you just merge some other branches into the master few minutes ago?

**Comment by dragonyanglong** [comment]
Thanks. I think no more rebase is needed now, since it shows no conflicts with the base branch.

**Comment by sbillinge** [comment]
Thanks @dragonyanglong

**Pull request merged**